backend/src/middleware/permissionsMiddleware.js

```diff
@@ -30,6 +33,14 @@ const isMyOwn = rule({
   return context.user.id === parent.id
 })
 
+const isMySocialMedia = rule({
+  cache: 'no_cache',
+})(async (_, args, { user }) => {
+  let socialMedia = await instance.find('SocialMedia', args.id)
+  socialMedia = await socialMedia.toJson()
+  return socialMedia.ownedBy.node.id === user.id
```

Comment:
Interesting! What is `socialMedia.ownedBy.node`? There are two different ids in Neo4J: It uses an internal `_id` which is an auto-incrementing number and we have our own custom `id` which is a random `uuid` in production environment.

How does `ownedBy.node` differ from `ownedBy.properties`?

Reply:
Honestly, at that point I was just `console.log`ing the JSON object in search for the right `id` – so I'm not quite sure what `ownedBy.node` is and haven't even heard of `ownedBy.properties`... I will check again!

I couldn't use the internal Neo4J `_id` because the frontend is sending _our_ id in the args and that's the one I need to compare to.